documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2118,6 +2151,7 @@ To deploy a {prodname} Oracle connector, you install the {prodname} Oracle conne
 +
 NOTE: If you use the {prodname} Oracle connector with Oracle XStream, obtain the JDBC driver as part of the Oracle Instant Client package.
 For more information, see xref:obtaining-oracle-jdbc-driver-and-xstreams-api-files[].
+. Download the link:https://repo1.maven.org/maven2/com/oracle/database/xml/xdb/{ojdbc8-version}/xdb-{ojdbc8-version}.jar[XDB library for Oracle] from Maven Central and extract the downloaded file to the directory that contains the {prodname} Oracle connector JAR file.
```

Comment:
In your review request, you refer to changes that might be "needed for the product side", which I take to mean for use downstream. But this content is only in the Deployment topic that's conditionalized for community use. 

Do we need to include a reference to obtaining the XDB library in `/partials/modules/all-connectors/ref-deploy-oracle-kafka-connect-yaml.adoc` ([L10](https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/partials/modules/all-connectors/ref-deploy-oracle-kafka-connect-yaml.adoc#L10), and [L42](https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/partials/modules/all-connectors/ref-deploy-oracle-kafka-connect-yaml.adoc#L42))?

And probably a short topic in the Deployment overview in `oracle.adoc`, analogous to (and adjacent to)  _[Obtaining the Oracle JDBC driver_,](https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/pages/connectors/oracle.adoc#L2151) that explains why you might want to obtain the XDB library.

Reply:
That's correct @roldanbob.  Additionally the user will need to obtain the `xmlparserv2` dependency at 
```
<dependency>
  <groupId>com.oracle.database.xml</groupId>
  <artifactId>xmlparserv2</artifactId>
  <version>21.6.0.0</version>
</dependency>
```
I also added a blurb about the `-Djavax.xml.parsers.SAXParserFactory` argument required for XML support with Oracle, so we may need to rework how to include this elsewhere when enabling XML support.

Reply:
Agreed. A procedure module, e.g., _Enabling support for Oracle XML data types_.

As far as adding content to the downstream deployment instructions, how shall we handle that.  Want me to open a separate PR against DBZ-3605 to make those changes? Or I could wait until you merge this and then add the downstream version of the content?